scripts/kind-e2e/e2e.sh

```diff
@@ -327,6 +327,12 @@ function test_with_e2e_tests {
 
 function deploy_globalnet {
     for i in 2 3; do
+      if kubectl --context=cluster${i} -n $subm_ns get serviceaccount submariner-globalnet > /dev/null 2>&1; then
```

Comment:
shouldn't an existing deployment be caught at much earlier point? Why not only create this if it doesn't already exist? something like...

if ! kubectl --context=cluster${i} get serviceaccount submariner-globalnet; then

Reply:
Sorry, not able to understand your comment. 

Even if the deployment already exists, we still want to clear the existing deployment and redeploy with the updated/new config files/Image, isnt it?